templates/Replica1.client.vol.j2

```diff
@@ -10,6 +10,9 @@ end-volume
 volume {{ volname }}-dht
     type cluster/distribute
     subvolumes {{ dht_subvol|join(' ') }}
+{% if decommissioned != "" %}
+    option decommissioned-bricks {{ decommissioned }}
```

Comment:
If you don't mind, can you pls explain this?
- Like when a brick/subvol is decommissioned we'll be performing ~replace brick kind of operation and should wait for heal to complete right? (or)
- Is it just like we don't need (~data in) old bricks and they can be kinda replaced all together?

Reply:
https://github.com/kadalu/glusterfs/blob/devel/xlators/cluster/dht/src/dht-shared.c#L960 

this just makes the brick not get any new data. But reads will be served from the same brick. There is no 'heal', but rebalance operation which is required.